mkdocs.yml

```diff
@@ -15,6 +15,9 @@ extra:
     provider: google
     property: UA-343802-3
 
+plugins:
+  section-index: {}
```

Comment:
Be careful with this plugin. It can break the side TOC. If you do not already use it, do not add it.